lib/screens/home/profile/profile_page.dart

```diff
@@ -65,17 +78,26 @@ class _ProfilePageState extends State<ProfilePage> {
                     },
                   ),
                   SizedBox(
-                    height: 20,
+                    height: (currentUser.isServiceProvider == true)? 10: 0,
                   ),
-                  RaisedButton(
-                    child: Text("Add pet"),
+                  (currentUser.isServiceProvider == true)? RaisedButton(
+                    child: Text("Add Service"),
                     onPressed: () {
-                      Navigator.of(context).pushNamed(AddPetPage.routeName,
-                          arguments: Pet.empty());
+                      Navigator.of(context).pushNamed(AddServicePage.routeName,
+                          arguments: Service.empty());
                     },
+                  ): SizedBox(),
+                  SizedBox(
+                    height: (currentUser.isServiceProvider == true)? 10: 0,
                   ),
+                  (currentUser.isServiceProvider == true)? RaisedButton(
+                    child: Text("My Services"),
+                    onPressed: () {
+                      Navigator.of(context).pushNamed(MyServicesPage.routeName);
+                    },
+                  ): SizedBox(),
```

Comment:
Can you extract all the `Service` related widgets to a separate Widget and only show that widget if the currentUser is a service provider? *to avoid duplicated condition